<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QChar常用功能</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入字符串</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入单个字符</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editStr">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editChar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetChars">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>每个字符的Unicode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCharJude">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单个字符特性判断</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>其他测试和功能</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="btnCompare">
      <property name="text">
       <string>QChar比较和替换</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>清空文本框</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btnConvLatin1">
      <property name="text">
       <string>与Latin1的转换</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnConvUTF16">
      <property name="text">
       <string>与UTF-16的转换</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>153</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="chkDigit">
      <property name="text">
       <string>isDigit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkLetter">
      <property name="text">
       <string>isLetter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkLetterOrNumber">
      <property name="text">
       <string>isLetterOrNumber</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkUpper">
      <property name="text">
       <string>isUpper</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkLower">
      <property name="text">
       <string>isLower</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkMark">
      <property name="text">
       <string>isMark</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkSpace">
      <property name="text">
       <string>isSpace</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkSymbol">
      <property name="text">
       <string>isSymbol</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkPunct">
      <property name="text">
       <string>isPunct</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结果显示</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
